engineapi/backup_backing_image.go

```diff
@@ -243,7 +246,7 @@ func (m *BackupBackingImageMonitor) syncBackupStatusFromBackingImageManager() (c
 	m.backupBackingImageStatus.DeepCopyInto(&currentBackupStatus)
 	m.backupStatusLock.RUnlock()
 
-	backupBackingImageStatus, err = m.client.BackupStatus(m.backupBackingImageName)
+	backupBackingImageStatus, err = m.client.BackupStatus(m.backingImageName)
```

Comment:
Leave a comment to explain the change from `m.backupBackingImageName` to `m.backingImageName`?

Reply:
Modified. Thanks